`./mach install` should give a better error message when `./mach build` wasn't run before.

```sh
./mach clean
./mach install -p INSTALL
 [mach] Dependency checks ...
 [mach] Probing for cmake: Found
 [mach] Probing for ninja: Found

 [mach] ['cmake', '--install', 'build', '--config', 'Debug', '--prefix', 'INSTALL']
CMake Error: Error processing file: ~/hacl-packages/build/cmake_install.cmake
Traceback (most recent call last):
  File "~/hacl-packages/./mach", line 463, in <module>
    main()
  File "~/hacl-packages/./mach", line 459, in main
    args.func(args)
  File "~/hacl-packages/./mach", line 68, in install
    _install(prefix=args.prefix, config=args.config)
  File "~/hacl-packages/./mach", line 53, in _install
    subprocess.run(cmake_cmd, check=True)
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['cmake', '--install', 'build', '--config', 'Debug', '--prefix', 'INSTALL']' returned non-zero exit status 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -262,9 +262,9 @@ jobs:
       - name: Install
         run: |
           mkdir pkg-release
-          ./mach install -p $PWD/pkg-release -c Release
+          ./mach install -p $PWD/pkg-release
           mkdir pkg-debug
-          ./mach install -p $PWD/pkg-debug
+          ./mach install -p $PWD/pkg-debug --debug
 
       - name: Upload artifacts
         uses: ./.github/actions/upload_artifacts

--- a/.github/workflows/build_pull_request.yml
+++ b/.github/workflows/build_pull_request.yml
@@ -222,9 +222,9 @@ jobs:
       - name: Install
         run: |
           mkdir pkg-release
-          ./mach install -p $PWD/pkg-release -c Release
+          ./mach install -p $PWD/pkg-release
           mkdir pkg-debug
-          ./mach install -p $PWD/pkg-debug
+          ./mach install -p $PWD/pkg-debug --debug
 
       - name: Upload artifacts
         uses: ./.github/actions/upload_artifacts

--- a/mach
+++ b/mach
@@ -64,9 +64,6 @@ def _install(prefix=None, config="Release"):
     [
         argument("-p", "--prefix", help="The path prefix to install into.", type=str),
         argument("--debug", help="Install a debug build.", action="store_true"),
-        argument(
-            "--release", help="Install a release build. (default)", action="store_true"
-        ),
     ]
 )
 def install(args):

--- a/mach
+++ b/mach
@@ -9,13 +9,14 @@
 # The mach driver for HACL.
 
 
-from ast import arg
 import os
 import pathlib
 import re
 import shutil
 import subprocess
 import sys
+from ast import arg
+
 from tools.benchmark import run_benchmarks
 
 from tools.configure import Config
@@ -42,30 +43,45 @@ from tools.utils import (
 # === SUBCOMMANDS === #
 
 
-def _install(prefix=None, config=None):
-    configuration = "Debug"
-    if config:
-        configuration = config
-    cmake_cmd = ["cmake", "--install", "build", "--config", configuration]
+def _install(prefix=None, config="Release"):
+    cmake_cmd = ["cmake", "--install", "build", "--config", config]
     if prefix:
         cmake_cmd.extend(["--prefix", prefix])
     print(cmake_cmd)
-    subprocess.run(cmake_cmd, check=True)
+    if subprocess.call(cmake_cmd) == 1:
+        print("`cmake --install` returned with an error.")
+        if config == "Debug":
+            print(
+                "Please make sure to run `./mach build` before `./mach install --debug`."
+            )
+        else:
+            print(
+                "Please make sure to run `./mach build --release` before `./mach install`."
+            )
 
 
 @subcommand(
     [
         argument("-p", "--prefix", help="The path prefix to install into.", type=str),
+        argument("--debug", help="Install a debug build.", action="store_true"),
         argument(
-            "-c",
-            "--config",
-            help="The config to install, i.e. Debug or Release.",
-            type=str,
+            "--release", help="Install a release build. (default)", action="store_true"
         ),
     ]
 )
 def install(args):
-    _install(prefix=args.prefix, config=args.config)
+    if args.debug and args.release:
+        print("Can't use `--debug` and `--release` at the same time.")
+        exit(1)
+
+    # "Release" is the default in `_install`.
+    if args.debug:
+        # Thus, we only pass a "Debug" config when asked for ...
+        print("Note: You are installing a debug build.")
+        _install(prefix=args.prefix, config="Debug")
+    else:
+        # ... and stick to the default when not.
+        _install(prefix=args.prefix)
 
 
 @subcommand(
@@ -75,7 +91,11 @@ def install(args):
         argument("--test", help="Build and run tests.", action="store_true"),
         argument("--benchmarks", help="Build benchmarks.", action="store_true"),
         argument("--benchmark", help="Build and run benchmarks.", action="store_true"),
-        argument("--no-openssl", help="Don't build and run OpenSSL benchmarks.", action="store_true"),
+        argument(
+            "--no-openssl",
+            help="Don't build and run OpenSSL benchmarks.",
+            action="store_true",
+        ),
         argument("-r", "--release", help="Build in release mode.", action="store_true"),
         argument(
             "-a",

--- a/mach
+++ b/mach
@@ -67,10 +67,6 @@ def _install(prefix=None, config="Release"):
     ]
 )
 def install(args):
-    if args.debug and args.release:
-        print("Can't use `--debug` and `--release` at the same time.")
-        exit(1)
-
     # "Release" is the default in `_install`.
     if args.debug:
         # Thus, we only pass a "Debug" config when asked for ...

